indexing-hadoop/src/test/java/io/druid/indexer/BatchDeltaIngestionTest.java

```diff
@@ -340,13 +340,16 @@ private HadoopDruidIndexerConfig makeHadoopDruidIndexerConfig(Map<String, Object
         new HadoopIngestionSpec(
             new DataSchema(
                 "website",
-                new StringInputRowParser(
-                    new CSVParseSpec(
-                        new TimestampSpec("timestamp", "yyyyMMddHH", null),
-                        new DimensionsSpec(ImmutableList.of("host"), null, null),
-                        null,
-                        ImmutableList.of("timestamp", "host", "host2", "visited_num")
-                    )
+                MAPPER.convertValue(
```

Comment:
it does convert recursively, everything to Map. also, it wouldn't matter for this test as long as
```
mapper.convertValue(mapper.convertValue(InputRowParser parser, Map.class), InputRowParser.class)
```
produces the right thing. Specific changes for DataSchema serde are tested in DataSchemaTest .